.github/workflows/Dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM ghcr.io/precice/precice:latest
+ARG PRECICE_TAG=latest
+ARG from=precice/precice:${PRECICE_TAG}
+FROM $from
 
+ARG OPENFOAM_EXECUTABLE=openfoam2112
+ARG OPENFOAM_ADAPTER_REF=develop
 # Install OpenFOAM
-ENV OPENFOAM_VERSION=2112
-RUN wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | sudo bash
-RUN sudo apt-get -qq install openfoam${OPENFOAM_VERSION}-dev
+RUN apt-get update
+RUN wget -q -O - https://dl.openfoam.com/add-debian-repo.sh | bash
+RUN apt-get -qq install $OPENFOAM_EXECUTABLE-dev git
```

Comment:
I hope this `<name>-dev` package naming is the same also for the Foundation versions (please check).

Reply:
No, but currently we only plan to support that in the CI